src/api/files/service.js

```diff
@@ -53,6 +58,125 @@ export async function ingestFile(uploadPayload) {
   }
 }
 
+/**
+ * Accepts submissions into the forms-submission-api
+ * @param {SubmitPayload} submitPayload
+ */
+export async function submit(submitPayload) {
+  const { sessionId, retrievalKey, main, repeaters } = submitPayload
```

Comment:
just to check, you're only using this for logging purposes right? That's fine if so, just checking I've not missed anything.

Reply:
`sessionId` ? Yes it's just for logging for now. I'm hoping this `/submit` route will change soon to take `{ metadata, definition, state... }` like we've discussed, at which point `sessionId` will become more useful.